modules/lead/toolchain/rode.tf

```diff
@@ -0,0 +1,63 @@
+module "ca-issuer" {
+  source = "../../common/ca-issuer"
+
+  enabled          = var.enable_rode
+  name             = "rode"
+  namespace        = var.namespace
+  common_name      = var.root_zone_name
+  cert-manager-crd = var.crd_waiter
+}
+
+module "grafeas_certificate" {
+  source = "../../common/certificates"
+
+  enabled         = var.enable_rode
+  name            = "grafeas-cert"
+  namespace       = var.namespace
+  domain          = "grafeas-server"
+  issuer_name     = module.ca-issuer.name
+  certificate_crd = var.crd_waiter
+  wait_for_cert   = true
+}
+
+module "rode_certificate" {
+  source = "../../common/certificates"
+
+  enabled         = var.enable_rode
+  name            = "rode-cert"
+  namespace       = var.namespace
+  domain          = "rode"
+  issuer_name     = module.ca-issuer.name
+  certificate_crd = var.crd_waiter
+  wait_for_cert   = true
+}
+
+data "helm_repository" "liatrio-harbor" {
+  name = "liatrio-harbor"
+  url  =  "https://harbor.toolchain.lead.prod.liatr.io/chartrepo/public"
+}
+
+data "template_file" "rode_values" {
+  template = file("${path.module}/rode-values.tpl")
+
+  vars = {
+    iam_arn      = var.rode_service_account_arn
+    grafeas_cert = module.grafeas_certificate.cert_name
+    rode_cert    = module.rode_certificate.cert_name
+    ingress_hostname     = "rode.${module.toolchain_namespace.name}.${var.cluster}.${var.root_zone_name}
+  }
+}
+
+resource "helm_release" "rode" {
+  count      = var.enable_rode ? 1 : 0
+  repository = data.helm_repository.liatrio-harbor.metadata[0].name
```

Comment:
why is `.metadata[0]` needed here?  that data source doesn't have a `metadata` argument

Reply:
Other examples I found(like here in [artifactory.tf](https://github.com/liatrio/lead-terraform/blob/22b6af99946d1c689550973610a4f2547a5e11d1/modules/lead/toolchain/artifactory.tf#L126)) used this attribute which according to the documentation is exported to be the name of the helm repository: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/data-sources/repository#name.

I'm fine with updating this to `data.helm_repository.liatrio_harbor.name` to make it more clear, just want to keep it consistent with others I found.

Reply:
I guess I don't care that much since this data source is deprecated anyways and we'll be removing it when upgrading the helm provider.